test/quantization/test_backward_compatibility.py

```diff
@@ -68,9 +68,9 @@ def remove_prefix(text, prefix):
         qmodule_traced = torch.jit.load(traced_module_file)
 
         expected = torch.load(expected_file)
-        self.assertEqual(qmodule(input_tensor), expected, prec=prec)
```

Comment:
not sure how I missed this in the original PR, would have thought the assert would have caught it.